plugin/src/main/resources/seedJobGroovyScript.groovy

```diff
@@ -59,6 +61,8 @@ def createView(ItemGroup parent, String name, FilePath file) {
     if (parent instanceof ModifiableViewGroup) {
         ((ModifiableViewGroup) parent).addView(View.createViewFromXML(name, file.read()))
         createdViews++
+    } else if (parent == null) {
+        throw new RuntimeException("Parent is null. This happens when the parent folder could not get created, which can happen if a job with the name of the parent folder already exists. To solve this rename the job or the parent folder.")
```

Comment:
- Please stick to the 120 character line length limit.
- "if a job with the name of the parent folder already exists": This is only one of the possible reasons, maybe add "which can happen **for example**..." to avoid confusion in case there was another problem.
- If you add the null check the null-safe operator should be removed from `parent?` below.